docs/transactions.md

```diff
@@ -1377,12 +1377,28 @@ The account is not revoked until sufficient number of Trustees approve it.
 
 Gets all proposed but not approved accounts.
 
```

Comment:
Please add the information about the new keys (account counters) to the `KV Store` section.